keyboards/nomu30/nomu30.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This file should be removed (and submitted to `the-via/keyboards`)

Reply:
Removed 1378aa8